apps/src/aiTutor/interactionsApi.ts

```diff
@@ -0,0 +1,20 @@
+import Lab2Registry from '../lab2/Lab2Registry';
+import {getAuthenticityToken} from '../util/AuthenticityTokenStore';
+import {AITutorInteraction} from './types';
+
+export async function savePromptAndResponse(interactionData: AITutorInteraction) {
+  try {
+    await fetch('/ai_tutor_interactions', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRF-Token': await getAuthenticityToken(),
+      },
+      body: JSON.stringify(interactionData),
+    });
+  } catch (error) {
+    Lab2Registry.getInstance()
+      .getMetricsReporter()
+      .logError('AI Tutor Interaction failed to save.', error as Error);
```

Comment:
@molly-moen Is this the best way to do this? I'm fuzzy on how Lab2 is involved in my case (is it?) but it looks like the most consistent example to follow? 🤔

Reply:
Since Java lab is not in Lab2 I think the best way is to import it directly: [example import](https://github.com/code-dot-org/code-dot-org/blob/8fca707964d24fea553ecae8610dbe01835bf334/apps/src/blockly/utils.js#L5), [example usage](https://github.com/code-dot-org/code-dot-org/blob/8fca707964d24fea553ecae8610dbe01835bf334/apps/src/blockly/utils.js#L152)